code/modules/events/aurora_caelus.dm

```diff
@@ -32,6 +32,16 @@
 		if(A.area_flags & AREA_USES_STARLIGHT)
 			for(var/turf/open/space/S in A)
 				S.set_light(S.light_range * 3, S.light_power * 0.5)
+		if(istype(A, /area/station/service/kitchen))
+			for(var/turf/open/seymour in A)
+				seymour.set_light(1, 0.75)
+			if(prob(1) || SSevents.holidays?[APRIL_FOOLS])
+				for(var/obj/machinery/oven/steamed_hams in A)
+					steamed_hams.balloon_alert_to_viewers("oh ye gods!")
```

Comment:
```suggestion
					steamed_hams.balloon_alert_to_viewers("oh egads!")
```

Reply:
he says oh ye gods >:(

Reply:
"oh ye gods" is NOT a real phrase whereas "oh egads" is and also the Simpsons wiki disagrees with you